go/vm/test/integration_test.go

```diff
@@ -709,7 +710,7 @@ func TestNoReturnDataForCreate(t *testing.T) {
 						byte(geth.PUSH1), byte(0),
 						byte(test.returnInstruction)}
 
-					createInputBytes := rightPadBytes(createInputCode, 32)
+					createInputBytes := common.RightPadSlice(createInputCode, 32)
```

Comment:
I am not sure if there should be dependencies in the VM implementations to the CT.

Reply:
Right, this is not great. However, right now we do have multiple CT dependencies from the VM to the CT directory, in particular for the CT adaptor. On the other hand, we have CT to VM dependencies for interfaces and basic types.

We will eventually have to fix a dependency order and clean this up in a follow-up.